sdk/src/Services/S3/Custom/Util/_bcl/S3PostUploadSignedPolicy.cs

```diff
@@ -111,47 +111,16 @@ private static byte[] addConditionsToPolicy(string policy, Dictionary<string, st
 
                     if (!found)
                     {
-                        var newCondition = new JsonObject
-                        {
-                            [newCond.Key] = newCond.Value
-                        };
-                        jsonConditions.Add(newCondition);
-                    }
-                }
-            }
-
-            return Encoding.UTF8.GetBytes(json.ToJsonString().Trim());
-        }
-
-        private static byte[] addTokenToPolicy(string policy, string token)
-        {
-            var json = JsonNode.Parse(policy) as JsonObject;
-            var found = false;
-            var conditions = json["conditions"] as JsonArray;
-            if (conditions != null)
-            {
-                for (int i = 0; i < conditions.Count; i++)
-                {
-                    if (conditions[i] is JsonObject obj && obj.ContainsKey(S3Constants.PostFormDataSecurityToken))
-                    {
-                        obj[S3Constants.PostFormDataSecurityToken] = token;
-                        found = true;
+                        var jsonCond = new JsonData();
+                        jsonCond.SetJsonType(ThirdParty.Json.LitJson.JsonType.Object);
```

Comment:
Is `ThirdParty.Json.LitJson.JsonType` correct? Or should we be using the `System.Text.Json` equivalent?